src/FileJet.php

```diff
@@ -72,6 +72,25 @@ public function getExternalUrl(string $url, string $mutation = '')
         return "{$this->config->getPublicUrl()}/ext/{$mutation}?src={$this->sign($url)}";
     }
 
+    public function getDetentionUrl(string $fileId, int $expires, string $mutation = ''): DownloadInstruction
+    {
+        $requestParameters = ['fileId' => $this->normalizeId($fileId), 'expires' => $expires];
+
+        $customDomain = $this->config->getCustomDomain();
+        if ($customDomain) {
+            $requestParameters['customDomain'] = $customDomain;
+        }
+
+        $mutation = $this->resolveAutoMutation($mutation);
+        if ($mutation) {
+            $requestParameters['mutation'] = $mutation;
+        }
```

Comment:
This is duplicate of what we already have in `getPrivateUrl()`.

The only thing which differ from `getPrivateUrl()` is endpoint which you are calling eg. instead of `file.privateUrl` you are calling `file.detentionUrl` here.

Could you please reuse these lines in both methods?